.github/workflows/linux.yml

```diff
@@ -907,6 +907,9 @@ jobs:
             python3 -m pip install $ov_dev_wheel_name[mxnet,caffe,kaldi,onnx,tensorflow2,pytorch]
           popd
 
+          # To enable pytest parallel features
+          python3 -m pip install pytest-xdist[psutil]
```

Comment:
Shouldn't this be added to `requirements.txt` of tests? https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/requirements.txt

Reply:
I think no, it should be a part of CI, because user could run those tests without this plugin.

Reply:
To add or not add additional dependency we should answer what will not work without it?
imho this is user's specific tool and he should decide to install it or not, depends on his decision use it or not

btw, why we do not include CI system dependencies (git, ca-certificates) to install_dependencies.sh script then?